requirements/constraints.txt

```diff
@@ -97,5 +97,9 @@ py2neo<2022
 # Sphinx requires docutils<0.18. This pin can be removed once https://github.com/sphinx-doc/sphinx/issues/9777 is closed.
 docutils<0.18
 
-# Temporary constraint on openedx-calc. Latest openedx-calc also contains symmath, which may upend symmath package version
-openedx-calc==2.0.1
+# mistune is a dependency of m2r (which is a dependency of sphinxcontrib-openapi)
+# m2r fails to specify the version of mistune that it needs leading to the error message:
+# AttributeError: module 'mistune' has no attribute 'BlockGrammar'
+# https://github.com/miyakogi/m2r/issues/66
+mistune<2.0.0
```

Comment:
Good details here. Another detail that's good to include is the answer to the question "when can we remove this constraint?" (if someone is reading comments in constraints.txt, there's a good change they're looking to de-constrain something ;)

The comment above `docutils<0.18` is a good example of this.

Reply:
I'm not sure there is a clear answer to [this](https://github.com/miyakogi/m2r/issues/66).
Either [m2r locks mistune or m2r updates to mistune 2.0 API](https://github.com/miyakogi/m2r/pull/43).
Neither seems likely at the moment without some outside help.